packages/zosfiles/__tests__/__system__/methods/upload/Upload.system.test.ts

```diff
@@ -25,7 +25,7 @@ let testEnvironment: ITestEnvironment;
 let defaultSystem: ITestPropertiesSchema;
 let dsname: string;
 let ussname: string;
-const inputfile = "./packages/zosfiles/__tests__/__system__/api/methods/upload/testfiles/upload.txt";
+const inputfile = "./packages/zosfiles/__tests__/__system__/methods/upload/testfiles/upload.txt";
```

Comment:
Might be good to use `__dirname` more in test resource paths as was done in `Invoke.system.test.ts`, to make the paths more dynamic if folder structure changes in the future.